app/views/ips/_activation_notice.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="govuk-inset-text">
   <h4 class="govuk-heading-s">New IPs are not immediately available</h4>
   <p class="govuk-body">
-    You can configure your access points now, but new IPs will not be activated until the next day.
+    You can configure your authenticator(s) now, but new IPs will not be activated until the next day.
```

Comment:
Don't really want to get too much into terminology now but how did we decide on authenticators?
Wouldn't infrastructure be more descriptive to network administrators?

Reply:
Sure yeah